servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/LoadBalancedStreamingHttpConnection.java

```diff
@@ -44,8 +44,9 @@
 /**
  * Makes the wrapped {@link StreamingHttpConnection} aware of the {@link LoadBalancer}.
  */
-final class LoadBalancedStreamingHttpConnection implements FilterableStreamingHttpLoadBalancedConnection,
-                   ReservedStreamingHttpConnection, ReservableRequestConcurrencyController,
+public class LoadBalancedStreamingHttpConnection
```

Comment:
Instead of opening an implementation class with extra methods (like `unwrap/limiter`), have you considered adding `io.servicetalk.http.api.LoadBalancedStreamingHttpConnection` interface just to define a contract for the load-balanced connection. Implementations will be free to do whatever if they follow the contract without tightening to our impl. class.